app/javascript/src/components/Login.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Button, Checkbox, FormControlLabel, Box, TextField } from "@mui/material";
+import GoogleIcon from "@mui/icons-material/Google";
+import FacebookIcon from "@mui/icons-material/Facebook";
+
+const Login = (): React.ReactElement => {
+    // const [email, setEmail] = useState("");
+    // const [password, setPassword] = useState("");
+    // const [rememberMe, setRememberMe] = useState(false);
+
+    // const handleEmailChange = (e: React.FormEvent<HTMLInputElement>): void => {
+    //     const target = e.target as HTMLInputElement;
+    //     setEmail(target.value);
+    // };
+
+    // const handlePasswordChange = (e: ChangeEventHandler<HTMLInputElement>): any => {
+    //     const target = e.target as HTMLInputElement;
+    //     setPassword(target.value);
+    // };
+
+    // const handleRememberMeChange = (e: any) => {
+    //     setRememberMe(e.target.checked);
+    // };
+
+    const handleSubmit = (e: any): void => {
```

Comment:
You should avoid using `any`. Instead specify what the type of `e` should be.

Also, avoid using abbreviations. ~e~ -> `event`